src/main/kotlin/org/flaxo/plagiarism/graph2viz.kt

```diff
@@ -34,6 +35,14 @@ import kotlin.random.Random
 fun Graph.toViz(canvasWidth: Int = 800, canvasHeight: Int = 500): Viz =
         toViz(canvasWidth.toDouble(), canvasHeight.toDouble())
 
+private const val defaultArrowLengthX = 30.0
+private const val defaultArrowLengthY = 8.0
+
+private val lines = mutableListOf<Line>()
+private val arrowLines = mutableListOf<List<Line>>()
```

Comment:
I suppose we can use `Pair<Line, Line>` instead of `List<Line>` because we know the lists size already.

Reply:
Replaced the `List<Line>` list with `Pair<Line, Line>`, but now the list is located in the `Viz.refreshGraph` function